SPECS/protobuf/protobuf.spec

```diff
@@ -1,18 +1,30 @@
 %{!?python2_sitelib: %global python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 %{!?python3_sitelib: %global python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 
+# Switch "sources_generation" to 1 when running a package build to generate cached sources for regular builds.
```

Comment:
Do we do this for our local builds or is this something specific ?

Reply:
Answered offline, but in short: yup, it's for local builds.